### ISSUE TITLE
Torch Hub Model Names

### ISSUE THREAD

**Issue by MKowal2** [facebookresearch/pytorchvideo#22]
## ❓ Questions on how to use PyTorchVideo

I am trying to load every model from the model zoo one-by-one. However, there is no list of model names to use in the torch.hub.load() function. For example, I have from the tutorial 'slow_r50' is the model corresponding to the Slow R50 8x8 model, but I can't find how to load the Slow R50 4x16 model. More specifically, what is the string I use in the torch.hub.load() function? Another example is the I3D model. I have tried a number of different strings trying to load a pre-trained I3D model from torch hub but non work. I see in the hubconf.py a few different models, but not all of them.

Could you supply a list of strings for the torch.hub.load() function and their corresponding model zoo names?

Thanks!

**Comment by shinkyo0513** [comment]
By taking reference to the API and the implementation of I3D in PySlowFast, I got an I3D model as follows. 
`
i3d_model = create_resnet(
                        stem_conv_kernel_size=(5, 7, 7),
                        stage_conv_a_kernel_size=(
                                                    (3,1,1), 
                                                    [(3,1,1), (1,1,1)],
                                                    [(3,1,1), (1,1,1)],
                                                    [(1,1,1), (3,1,1)]
                                                 ),
                        head_pool_kernel_size=(8, 7, 7)
                    )
`
It can load the pre-trained parameters of I3D provided in the model zoo of PyTorchVideo. But the top-1 accuracy on my Kinetics400 validation set is only 64.449.

**Comment by kalyanvasudev** [comment]
@lyttonhao , could you please take look at this issue?

**Comment by lyttonhao** [comment]
@MKowal2 Thanks for the raising this point! We will update the documents to provide the summary of supported torch hub models.

@shinkyo0513 The model zoo accuracy is measured on the testing setting of  10 x 3 views. It will have differences with the single-crop testing. Meanwhile, we will also release the I3D torch hub models soon.

**Comment by lyttonhao** [comment]
Please check [here](https://github.com/facebookresearch/pytorchvideo/tree/master/pytorchvideo/models/hub) for the summarized full list of hub models.
